src/ingress/views.py

```diff
@@ -20,6 +23,5 @@ def post(self, request, queue):
         if result:
             return HttpResponse(status=200)
         else:
-            # TODO: improve error reporting. When does this actually need to fail?
```

Comment:
De regels `logger.error()` en `return HttpResponse` hieronder hoeven niet in de else, want de if returned al direct.